requirements.txt

```diff
@@ -1,7 +1,17 @@
+# Direct dependencies
```

Comment:
It looks like all of the direct dependencies are a part of the ignore... so what is this checking for dependencies? Just the testing libraries?